From 5b28f3488a033bd38b31071463b112df1713b64b Mon Sep 17 00:00:00 2001
From: Thorsten Behrens <tbehrens@suse.com>
Date: Wed, 15 Feb 2012 16:51:52 +0100
Subject: [PATCH] Fix fdo#45177 - avoid linked undo for the while.

While I try to wrap head around subtleties for linked undo in
Impress outline view, disable that here for the while. Better than
a crash for sure.
(cherry picked from commit 57e6f04066cfddfa79eb2b79cf5c45653c79eff3)

Conflicts:

	sd/source/ui/view/outlview.cxx
---
 sd/source/ui/view/outlview.cxx |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/sd/source/ui/view/outlview.cxx b/sd/source/ui/view/outlview.cxx
index 53ebbb4..cdd442d 100644
--- a/sd/source/ui/view/outlview.cxx
+++ b/sd/source/ui/view/outlview.cxx
@@ -1672,12 +1672,6 @@ void OutlineView::EndModelChange()
 
     DBG_ASSERT( bHasUndoActions == (mpOutliner->GetUndoManager().GetUndoActionCount() != 0), "sd::OutlineView::EndModelChange(), undo actions not in sync!" );
 
-    if( bHasUndoActions )
-    {
-        SfxLinkUndoAction* pLink = new SfxLinkUndoAction(pDocUndoMgr);
-        mpOutliner->GetUndoManager().AddUndoAction(pLink);
-    }
-
     mpOutliner->GetUndoManager().LeaveListAction();
 
     if( bHasUndoActions && mpOutliner->GetEditEngine().HasTriedMergeOnLastAddUndo() )
-- 
1.7.7.6

